src/chainstate/coordinator/mod.rs

```diff
@@ -77,6 +94,26 @@ pub enum PoxAnchorBlockStatus {
     NotSelected,
 }
 
+#[derive(Debug)]
+pub struct BlockExitRewardCycleInfo {
+    // The block id that corresponds to this exit cycle information
+    pub block_id: StacksBlockId,
+    // The block id of the parent
+    pub parent_block_id: StacksBlockId,
+    // The reward cycle of the block that corresponds to this exit cycle information
+    pub block_reward_cycle: u64,
+    // The stacks block "height" within the reward cycle that this block belongs to
```

Comment:
Does this mean that this value varies between 0 and 2100?

Reply:
Yes it does